content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -0,0 +1,303 @@
+wait a moment for the newly defined RBAC policy to become effective.
+
+1.  Enable the Istio RBAC for the namespace:
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -n $NS -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: RbacConfig
+    metadata:
+      name: default
+    spec:
+      mode: 'ON_WITH_INCLUSION'
+      inclusion:
+        namespaces: ["rbac-groups-test-ns"]
+    EOF
+    {{< /text >}}
+
+1.  Run the following command to verify that the curl connection
+to the httpbin service is rejected after the RBAC policy has become effective.
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
+    {{< /text >}}
+
+    When the RBAC policy has become effective, it should return HTTP code 403:
+
+    {{< text plain >}}
+    403
+    {{< /text >}}
+
+1.  Create a service role with name `httpbin-viewer` to give read access to
+the `httpbin` service:
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -n $NS -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: ServiceRole
+    metadata:
+      name: httpbin-viewer
+      namespace: rbac-groups-test-ns
+    spec:
+      rules:
+      - services: ["httpbin.rbac-groups-test-ns.svc.cluster.local"]
+        methods: ["GET"]
+    EOF
+    {{< /text >}}
+
+1.  Create a service role binding with name `bind-httpbin-viewer` that
+assigns the `httpbin-viewer` role to users in the `group1`.
+Wait a moment for the newly defined RBAC policy to become effective.
+Currently `request.auth.claims` is the only property that supports matching
+against a list of strings in the JWT token.
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -n $NS -f -
+    apiVersion: "rbac.istio.io/v1alpha1"
+    kind: ServiceRoleBinding
+    metadata:
+      name: bind-httpbin-viewer
+      namespace: rbac-groups-test-ns
+    spec:
+      subjects:
+      - properties:
+          request.auth.claims[groups]: "group1"
+      roleRef:
+        kind: ServiceRole
+        name: "httpbin-viewer"
+    EOF
+    {{< /text >}}
+
+1.  Verify that the connection to the `httpbin`
+service succeeds after the RBAC policy has become effective.
+
+    {{< text bash >}}
+    $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
+    200
+    {{< /text >}}
+
+    When the http header includes a valid JWT with the `groups` claim
+    value [`"group1"`, `"group2"`]
+    containing `group1`, it should return HTTP code 200:
+
+    {{< text plain >}}
+    200
+    {{< /text >}}
+
+## Configure the authorization of list-typed claims
+
+Istio RBAC also supports configuring the authorization of list-typed claims.
```

Comment:
Get rid of the word "also". It is considered a word with very little semantic value, in other words, it does not add any significant meaning:
Istio RBAC supports ...

Reply:
Done.